subcommands/targets/show.go

```diff
@@ -42,6 +44,31 @@ func init() {
 	}
 	showCmd.AddCommand(showAppCmd)
 	showAppCmd.Flags().Bool("manifest", false, "Show an app docker manifest")
+
+	sbomCmd := &cobra.Command{
+		Use:   "sboms <version> [<build> <artifact>]",
+		Short: "Show SBOMs for a specific target.",
+		Run:   doShowSboms,
+		Args:  cobra.RangeArgs(1, 3),
+		Example: `
+  # Show all SBOM files for Target version 42:
+  fioctl targets show sboms 42
+  
+  # Download all SBOM files for Target version 42:
+  fioctl targets show sboms 42 --download /tmp
+
+  # Show an overview of an SBOM from version 42:
+  # File with .spdx.tar.zst are produced by Yocto. Yocto produces an SPDX file
+  # per package. This will go through all the files and show the packages.
+  fioctl targets show sboms 42 42/raspberrypi4-64 initramfs-ostree-lmp-image-raspberrypi4-64.spdx.tar.zst
+  
+  # Show an overview of a container's SBOM from version 42:
+  # Files like this are from container builds where Syft was used
+  fioctl targets show sboms 42 41/build-armhf alpine:latest/arm.spdx.json`,
```

Comment:
This `41/build-armhf` seams to be a typo and on 41 it should be 42

Reply:
no. 42 in this case was Platform build that inherited its containers from build 41

Reply:
right, thanks and sorry for the noise